src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx

```diff
@@ -31,7 +31,7 @@ Distributed tracing tracks and observes service requests as they flow through di
 
 Distributed tracing systems collect data as the requests go from one service to another, recording each segment of the journey as a span. These spans contain important details about each segment of the request and are combined into one trace. The completed trace gives you a picture of the entire request.
 
-  ![Diagram showing how spans are generated in a distributed trace](./images/intro-span.svg "Diagram showing how spans are generated in a distributed trace")
+  ![Diagram showing how spans are generated in a distributed trace](./images/intro-DT.png "Diagram showing how spans are generated in a distributed trace")
 
   <figcaption>
     Here is an example a web transaction where agents measure the time spent in each service. Agents then send that timing information to New Relic as spans where they are combined into one distributed trace.
```

Comment:
Screenshot looks fine in my localhost, both in light mode and dark mode. Thanks! I wonder if the caption needs a comma between "spans" and "where"? Feel free to add the suggested comma or ignore, then go ahead and merge your PR. Thanks!